### PR TITLE
refactor: convert dash-case md-autosize to camelCase

### PR DESCRIPTION
**Pull request by EladBezalel**

### CODE REVIEW

**Comment by jelbourn** [comment]
LGTM, thanks

**Comment by jelbourn** [comment]
@EladBezalel I also just realized that the properties should also be renamed to `mdAutosizeMinRows` and `mdAutosizeMinRows`

**Comment by EladBezalel** [comment]
@jelbourn seems like #2432 already fixed it.. should i just change to `mdAutosizeMinRows` and `mdAutosizeMaxRows`?

**Comment by jelbourn** [comment]
Yeah, that would be good; we'd still need to support the old names for one release as well (via getter / setter)

**Comment by EladBezalel** [comment]
@jelbourn

File: src/lib/input/autosize.ts [hunk 1]
```diff
@@ -17,9 +17,25 @@ export class MdTextareaAutosize implements OnInit {
   /** Minimum number of rows for this textarea. */
   @Input() minRows: number;
 
+  get mdAutosizeMinRows(): number {
+    return this.minRows;
+  }
+
+  @Input() set mdAutosizeMinRows(value: number) {
```
Comment (jelbourn) [inline src/lib/input/autosize.ts]: `@Input()` should go on the `getter`. We should add that to the coding standards...

File: src/lib/input/autosize.ts [hunk 2]
```diff
@@ -17,9 +17,25 @@ export class MdTextareaAutosize implements OnInit {
...
+  }
+
   /** Maximum number of rows for this textarea. */
   @Input() maxRows: number;
 
+  get mdAutosizeMaxRows(): number {
+    return this.maxRows;
+  }
+
+  @Input() set mdAutosizeMaxRows(value: number) {
```
Comment (jelbourn) [inline src/lib/input/autosize.ts]: The unit test needs to be updated as well.

Comment (EladBezalel) [inline src/lib/input/autosize.ts]: do you mean adding another part that checks that the old and the new api works?

Comment (jelbourn) [inline src/lib/input/autosize.ts]: The test component in the spec still has `minRows` and `maxRows`. It should use the updated API.

**Pull request merged**